### PR TITLE
 fix: Check for handle on login

### PR DESCRIPTION
**Pull request by Yserz**

### CODE REVIEW

File: app/script/auth/page/Login.js [hunk 1]
```diff
@@ -125,11 +126,20 @@ class Login extends React.PureComponent {
 
   componentWillReceiveProps = nextProps => this.readAndUpdateParamsFromUrl(nextProps);
 
+  navigateChooseHandleOrWebapp = () => {
+    return Promise.resolve().then(
```
Comment (AndyLnd) [inline app/script/auth/page/Login.js]: no Promise needed here

File: app/script/auth/page/Login.js [hunk 2]
```diff
@@ -125,11 +126,20 @@ class Login extends React.PureComponent {
...
+          : this.props.history.push(ROUTE.CHOOSE_HANDLE)
+    );
+  };
+
   immediateLogin = () => {
     return Promise.resolve()
       .then(() => this.props.doInit({isImmediateLogin: true}))
       .then(() => this.props.doInitializeClient(true, undefined))
-      .then(() => window.location.replace(URLUtil.pathWithParams(EXTERNAL_ROUTE.WEBAPP)))
+      .then(() => this.navigateChooseHandleOrWebapp())
```
Comment (AndyLnd) [inline app/script/auth/page/Login.js]: no need to create a new inline function

File: app/script/auth/page/Login.js [hunk 3]
```diff
@@ -174,7 +184,7 @@ class Login extends React.PureComponent {
           ? this.props.doLoginAndJoin(login, this.state.conversationKey, this.state.conversationCode)
           : this.props.doLogin(login);
       })
-      .then(() => window.location.replace(URLUtil.pathWithParams(EXTERNAL_ROUTE.WEBAPP)))
+      .then(() => this.navigateChooseHandleOrWebapp())
```
Comment (AndyLnd) [inline app/script/auth/page/Login.js]: see above

**Comment by gregor** [comment]
I reverted this PR in order to avoid impact on release

**Pull request merged**